packages/edit-post/src/style.scss

```diff
@@ -104,13 +104,56 @@ body.block-editor-page {
 .wp-block {
 	max-width: $content-width;
 
-	&[data-align="wide"],
-	&.alignwide {
+	&[data-align="wide"] {
 		max-width: 1100px;
 	}
 
-	&[data-align="full"],
-	&.alignfull {
+	&[data-align="full"] {
 		max-width: none;
 	}
 }
+
+.wp-align-wrapper {
+	max-width: $content-width;
+	margin-left: auto;
+	margin-right: auto;
+
+	> .wp-block,
```

Comment:
why do we need this?

Reply:
Because the max-width should apply to the align wrapper, but not the nested block